cmd/kube-apiserver/app/aggregator.go

```diff
@@ -288,6 +288,7 @@ var apiVersionPriorities = map[schema.GroupVersion]priority{
 	{Group: "node.k8s.io", Version: "v1beta1"}:                  {group: 16300, version: 9},
 	{Group: "discovery.k8s.io", Version: "v1beta1"}:             {group: 16200, version: 12},
 	{Group: "discovery.k8s.io", Version: "v1alpha1"}:            {group: 16200, version: 9},
+	{Group: "flowcontrol.apiserver.k8s.io", Version: "v1alpha1"}:{group: 16100, version: 9},
```

Comment:
it doesnt honor runtime config

Reply:
you mean, the problem is that this line should be conditional on runtime config because this group is disabled by default, right?

Reply:
This looks correct to me. If runtime-config does not enable the API group, this line is inert